streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -154,9 +154,9 @@ final void transitionTo(final Task.State newState) {
     }
 
     @Override
-    public void update(final Set<TopicPartition> topicPartitions, final Map<String, List<String>> nodeToSourceTopics) {
```

Comment:
Since the root cause of this bug was basically just confusion over what exactly this set contains, a renaming feels in order